exporter/README.md

```diff
@@ -79,6 +79,7 @@ service:
 
 When multiple exporters are configured to send the same data (e.g. by configuring multiple
 exporters for the same pipeline) the exporters will have a shared access to the data.
+If the exporter signals that it mutates data via its Capabilities, a copy is actually passed to the exporter.
```

Comment:
Need to fix the previous line correct?

Reply:
@bogdandrutu , think we should add additional line for this(which is my fix now),please let me know if any suggestions . Thank you.

Reply:
@bogdandrutu This is my first time on contributing Open source, Feel free to help take a look at the changes, Thank you so much!

Reply:
@codeboten Can you help review this also? thank you so much!

Reply:
I am stickying with my position, that previous sentence and next sentence need to be modified accordingly since now we can have exporters that mutate the data.

Reply:
@bogdandrutu @codeboten  I have updated the sentence per the suggestions, can you help review again, thank you so much!